api/README.md

```diff
@@ -4,6 +4,6 @@
 2. Activate virtual environment: `. venv/bin/activate` (if you're using bash)
 3. Install dependencies: `pip3 install -r requirements.txt`
 4. Set up database in docker: `./mongo_setup.sh`
-5. Run: `uvicorn main:app --reload`
+5. Run: `sh -c 'cd .. && uvicorn api.main:app --reload'`
```

Comment:
This change is important!!! Make sure to use this new command whenever you run the backend.